src/App.js

```diff
@@ -22,8 +25,15 @@ class App extends React.Component {
     lastPath: '',
     path: '',
     search: '',
-    sidePanelOpened: false,
     notifications,
+    // signerOpened: false,
+    // web3Action: {},
+    signerOpened: true,
+    web3Action: {
+      error: null,
+      intent: actionIntent,
+      paths: actionPaths,
+    },
```

Comment:
I've left this here to force the signer open when the app starts so it's easy to demo, but I'll remove it before we merge.